samples/wifi/raw_tx_packet/src/main.c

```diff
@@ -125,6 +125,27 @@ static void wifi_set_mode(int mode_val)
 	}
 }
 
+#ifdef CONFIG_RAW_TX_PACKET_SAMPLE_TX_INJECTION_MODE
+static int wifi_set_tx_injection_mode(void)
+{
+	struct net_if *iface;
+
+	iface = net_if_get_first_wifi();
+	if (!iface) {
+		LOG_ERR("Failed to get Wi-Fi iface");
+		return -1;
```

Comment:
If i see the code below - i think you are invoking the API like this:
	if (wifi_set_tx_injection_mode()) {
		return -1;
	}
so in the above case, if return is -1, the code will not think it is a failure. Kindly check this.
Also, if there is a pre-defined return value - please return that, instead of -1 and 0.